R/learner_partykit_surv_cforest.R

```diff
@@ -112,11 +114,14 @@ LearnerSurvCForest = R6Class("LearnerSurvCForest",
       ps$values$mincriterion = 0
       ps$values$saveinfo = FALSE
 
+      # set cores to 1
+      ps$values$cores = 1
```

Comment:
it's now also possible to do `p_int(..., init = 1)` but this is also fine

Reply:
ha, new syntax, nice! to be consistent with eveyr other learner for now let's leave it as is?